Source/SwiftLintFramework/Rules/TrailingCommaRule.swift

```diff
@@ -99,6 +102,22 @@ public struct TrailingCommaRule: ASTRule, ConfigurationProviderRule {
             )
         ]
     }
+
+    private func trailingCommaIndex(_ contents: String, file: File, offset: Int) -> Int? {
+        // swiftlint:disable:next force_try
+        let regex = try! NSRegularExpression(pattern: ",", options: [.ignoreMetacharacters])
+        let range = NSRange(location: 0, length: contents.characters.count)
```

Comment:
`contents.characters.count` is not always equal to `(contents as NSString).length`